### DISCUSSION TITLE
Clarification on the CDM tables being loaded

### DISCUSSION THREAD

**Discussion by dsobolew** [OHDSI/ETL-Synthea#148] [General]
Hi, I've been looking into using this package and have a few questions on the CDM tables being used that I can't find answers to in the documentation. I see in the package details that quite a few CDM tables are created in the CreateCDMTables function and never populated with data from the Synthea or Athena CSV files. For instance COHORT and NOTE are tables that are created and never populated. Is this intentional? Also, I see Synthea tables are created and loaded (such as @synthea_schema.organizations) and never incorporated into the final OMOP CDM.

Is there a specific reason to include these empty or unused tables in the SQL DB? Are there future plans to incorporate these somehow?

Thanks!

**Comment by fdefalco** [comment]
The COHORT table is used to store the result of cohort generation scripts, so no content is populated as part of loading the data. 
I don't believe there are simulated physician notes in Synthea, which is what would go in the NOTE table.

Data generated by Synthea that has not yet been incorporated into ETL Synthea are candidates for new features if there are use cases you are looking to support, please let us know.
